### DISCUSSION TITLE
en/chapter_data_structure/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#1514] [Announcements]
# en/chapter_data_structure/

Data Structures and Algorithms Crash Course with Animated Illustrations and Off-the-Shelf Code

https://www.hello-algo.com/en/chapter_data_structure/

**Comment by ly3638** [comment]
补码1000 0000 的反码是1000 0001 然后原码是1111 1110 这样吗

**Reply by ly3638** [reply]
为什么没有回答我的问题
